WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -282,6 +282,9 @@ class MediaCoordinator: NSObject {
     func cancelUploadAndDeleteMedia(_ media: Media) {
         cancelUpload(of: media)
         delete(media: [media])
```

Comment:
I just noticed that ` delete(media: [media])` calls `cancelUpload(of: media)`

https://github.com/wordpress-mobile/WordPress-iOS/blob/cfee33ebc58eb0ccef2d35c93d9e420948caae83/WordPress/Classes/Services/MediaCoordinator.swift#L322-L323

Reply:
Yeah, you are right. It is a no-op when called second time, so it should be OK. I left a note to address it separately.